shared/schemas/workflow-result-schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "WorkflowResult",
+    "description": "Result of a workflow execution",
+    "type": "object",
+    "properties": {
+        "completedWith": {
+            "description": "The state of workflow completion.",
+            "type": "string",
+            "enum": [
+                "error",
+                "success"
+            ]
+        },
+        "message": {
+            "description": "High-level summary of the current status, free-form text, human readable.",
+            "type": "string"
+        },
+        "nextWorkflows": {
+            "description": "List of workflows suggested to run next. Items at lower indexes are of higher priority.",
+            "type": "array",
+            "items": {
+                "$ref": "workflow-ref-schema.json"
+            }
+        },
+        "outputs": {
+            "description": "Additional structured output of workflow processing. This can contain identifiers of created resources, links to resources, logs or other output.",
+            "type": "array",
+            "items": {
+                "$ref": "workflow-result-item-schema.json"
+            }
+        }
+    },
+    "required": [
+        "completedWith"
```

Comment:
I'm not sure about having this as required. My concern is about being too enforcing.

Reply:
ok, removing